docs/maatregelen/index.md

```diff
@@ -1,71 +1,80 @@
 ---
 title: Maatregelen
+summary: Overzicht van aanbevolen maatregelen voor overheden die algoritmes of AI ontwikkelen of gebruiken. De maatregelen helpen om te voldoen aan de eisen.
 hide:
 - navigation
 ---
-
-De maatregelen die zijn opgenomen in het Algoritmekader geven aan 'hoe' overheidsorganisaties invulling *kunnen* geven aan specifieke vereisten.
-Deze laag is aan het Algoritmekader toegevoegd, omdat het organisaties handvatten kan geven om zelf op een betekenisvolle wijze invulling te geven aan de vereisten.
-In het Algoritmekader zijn al deze maatregelen gekoppeld aan de levenscyclus van een algoritme, vereisten en rollen.
-Daarmee is inzichtelijk wanneer en met wie deze maatregelen logischerwijs in stelling kunnen worden gebracht.
-Organisaties zijn niet verplicht om deze maatregelen te volgen, hoewel ze soms dermate sterk aan de vereisten zijn gekoppeld of voortkomen uit standaarden dat ze praktisch gezien wel moeten worden toegepast.
-
-Een maatregel kan bijvoorbeeld zijn:
-
-> 'onderzoek het ontwikkelde algoritme op onbewuste vooringenomenheid (discriminatie) door middel van een bias-analyse'.
```

Comment:
we zijn met deze maatregel nog bezig, die staat er nu nog niet in. Tzt kijken of dit nog overeenkomt? Of hier een ander voorbeeld nemen die nu al inhet kader staat?

Reply:
Heb je een suggestie voor een ander voorbeeld? Inhoudelijk zit ik er niet zo in :)